docs(start-here,core-concepts,development,import-models,): restructure core-concepts/model (#181)
instill-ai/instill.tech: src/pages/docs/core-concepts/model.mdx

Besides, the [Python Backend](https://github.com/triton-inference-server/python_backend) enables Triton to support any model written in Python.
To make your models VDP-ready, please refer to [Prepare models](/docs/prepare-models/overview).

## Definition

VDP uses [ModelDefinition](https://github.com/instill-ai/protobufs/blob/main/vdp/model/v1alpha/model_definition.proto#L30-L76) to define how to configure and import a model from a supported model source.
Please check out [Import Models](/docs/import-models/overview) to learn more.

## Model Instance

A **Model Instance** represents a tagged snapshot of a Model. A model may have multiple model instances.
The tag of a model instance depends on the model source and what versioning control tool the model source uses.

_VDP captures the versioning of your models via model instances._
During model development, ML teams version models to track and manage changes and collaborate between members.
As you use VDP to import your model, different versions of your model are also imported as tagged model instances.
In this case, you can keep a single history of your models and the original versioning serves as the single source of truth.

> Sometimes we use the two terms _model_ and _model instance_ interchangeably.

Reviewer: ```suggestion
Throughout the documentation, we use the two terms _model_ and _model instance_ interchangeably.
```

> For example, we say that model is the core component of data pipeline to convert the unstructured visual data to structured insights.

Reviewer: ```suggestion
For example, when we say that **Model** is the core component of data pipeline to convert the unstructured visual data to structured insights, what actually happens underneath is the model instances being used to construct a data pipeline.
```

> This is a high level concept, but underneath model instances are used to construct a data pipeline.

Reviewer: ```suggestion
```

> The examples below showcase pipeline recipes that incorporate single or multiple model instances.

Reviewer: ```suggestion
A VDP pipeline can have multiple model instances. The examples below showcase pipeline recipes that incorporate single or multiple model instances.
```

In which `http://localhost:8083` is the default URL of the VDP model backend. `{model-id}` and `{instance-id}` correspond to the ID of model and model instance respectively.

> ## Release stage

Reviewer: Release stage has been moved to `core-concepts/overview`. Components can share that section directly.

Author: updated
